app/controllers/room_controller.rb

```diff
@@ -0,0 +1,9 @@
+class RoomsController < ApplicationRecord
+  def index
+    @rooms = Room.all
+  end
+
+  def show
```

Comment:
Because there isn't a user story that deals with showing a single room, we wouldn't need this `show` action.